policy/resolver.go

```diff
@@ -1264,7 +1281,9 @@ func (s *LocalServices) jobsToQueries(ctx context.Context, policyMrn string, cac
 
 		cache.executionQueries[checksum] = executionQuery
 		executionJob.Queries[codeID] = executionQuery
-
+		// TODO: unsure if this needs to go in here.. some code uses the executionJob.Queries for different assertions
+		// so maybe we want it in here as well?
+		executionJob.Queries[query.Mrn] = executionQuery
```

Comment:
unsure if we want this here. it seems to work just fine with it and i've seen some code that iterates over that map of queries.